components/report-manager-minio-gateway/habitat/hooks/health-check

```diff
@@ -0,0 +1,19 @@
+#!{{pkgPathFor "core/bash"}}/bin/bash
+
+exec 2>&1
+
+#curlOpts="-sS --fail --max-time 2"
```

Comment:
Is there a reason why we are keeping these commented lines?

Reply:
@kalroy  These are just to the reminders of pending work to me.